src/components/structures/auth/ForgotPassword.js

```diff
@@ -194,9 +194,16 @@ module.exports = React.createClass({
             serverName: this.props.serverConfig.hsName,
         });
         if (this.props.serverConfig.hsNameIsDifferent) {
-            // TODO: TravisR - Use tooltip to underline
+            const TextWithTooltip = sdk.getComponent("elements.TextWithTooltip");
+
             yourMatrixAccountText = _t('Your Matrix account on <underlinedServerName />', {}, {
-                'underlinedServerName': () => <u>{this.props.serverConfig.hsName}</u>,
+                'underlinedServerName': () => {
+                    return <TextWithTooltip
+                        class="mx_Login_underlinedServerName"
+                        tooltip={this.props.serverConfig.hsUrl}>
```

Comment:
Maybe end bracket on next line? (It seems strange to see the props and child content with same indent and nothing breaking them up visually...)